resources/function_help/json/make_circle

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "make_circle",
+  "type": "function",
+  "description": "Creates a polygon geometry from a circle.",
```

Comment:
I wonder if "creating polygon from a circle" description really explains what it's about. From what I understand (and I may be wrong), it actually helps to create a circular polygon geometry (not sure for the grammatical construction). I don't feel that the current description shows that the result is a circle.